user_guide.adoc

```diff
@@ -1167,25 +1249,190 @@ i.e. two least significant bits of the first byte would be lost and
 not parsed due to alignment.
 ====
 
-Last, but not least, note that it's also possible to parse bit-packed
-integers using old-school methods with value instances. Here's the
-very first example with IPv4 packed start, unpacked manually:
+[[bit-ints-le]]
+==== Little-endian order
+
+IMPORTANT: Feature available since v0.9.
+
+Most formats using little-endian _byte order_ with packed multi-byte
+bit fields (e.g. http://formats.kaitai.io/android_img/[android_img],
+http://formats.kaitai.io/rar/[rar] or http://formats.kaitai.io/swf/[swf])
```

Comment:
Minor thing - could you change these URLs to use HTTPS? Or if Asciidoc supports it, it would be even better to use URLs without an explicit protocol, like `//formats.kaitai.io/android_img/`, so that the link uses the same protocol that the current page was accessed with (HTTP or HTTPS).

Reply:
OK, makes sense. However, as there are many such links using explicit protocol throughout the docs, I'll find them with fulltext search and fix them all (but outside of this PR), that'll be probably easier.